sources/packages/backend/apps/api/src/services/institution-location/institution-location.service.ts

```diff
@@ -212,6 +212,25 @@ export class InstitutionLocationService extends RecordDataModelService<Instituti
     return found.length === locations.length;
   }
 
+  /**
+   * Validate if location code is unique for the institution.
+   * @param institutionId
+   * @param locationCode
+   * @returns whether there is already a location code for the institution.
+   */
+  async validateLocationCodeIsUniqueForInstitution(
```

Comment:
Can this method be changed to follow the same naming convention and Typeorm query (object format) from the `hasExistingProgramWithSameSABCCode`?
The name would be something like `hasLocationCodeForInstitution` or something like this.

Reply:
ok. Done. Thanks!